examples/src/main/java/org/apache/spark/examples/sql/JavaSQLDataSourceExample.java

```diff
@@ -389,6 +392,67 @@ private static void runCsvDatasetExample(SparkSession spark) {
     // $example off:csv_dataset$
   }
 
+  private static void runTextDatasetExample(SparkSession spark) {
+    // $example on:text_dataset$
+    // A text dataset is pointed to by path.
+    // The path can be either a single text file or a directory of text files
+    String path = "examples/src/main/resources/people.text";
```

Comment:
`people.text` -> `people.txt`

Reply:
Oh... sorry I manually tested in my local with create the new one, but forget to rename. Thanks!